docs/references/go/other-examples.mdx

```diff
@@ -5,30 +5,72 @@ description: Explore different examples of how to utilize Clerk with Go.
 
 # Examples
 
-Below is a code snippet showing what you can do from your backend with Clerk. Look at the [API reference](/docs/references/backend/overview) for a full list of requests you can make from your backend.
+Let's see an example that calls a few different [Clerk Backend API](/docs/references/backend/overview) operations using the Clerk Go SDK.
 
-## Update metadata
+In the snippet below, you will create an organization and update its slug. You'll fetch all organization
+memberships, loop through them to get the first one, and get more details about the organization's user.
 
-Add a new key/value to the private metadata stored on a user.
+Your Clerk secret key is required. If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
```

Comment:
```suggestion
<Callout type="info">

Your Clerk secret key is required. If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.

</Callout>
```

This feels appropriate for a callout to me

Reply:
:+1: Updated.